### ISSUE TITLE
TypeError: slice indices must be integers or None or have an __index__ method

### ISSUE THREAD

**Issue by crazygonuts** [hako-mikan/sd-webui-regional-prompter#88]
As of commit cf8c54b I am getting this when attempting to use regional prompter:

```
['(best quality, masterpiece:1.4) ,  a ww1 battlefield BREAK (best quality, masterpiece:1.4) ,  a battlemech BREAK (best quality, masterpiece:1.4) ,  a sherman tank'] ['(worst quality, low quality, normal quality:2),  neg-anime, neg-sketch-2']
pos tokens : [[9, 8, 8]], neg tokens : [[12]]
before_process_batch :  (best quality, masterpiece:1.4) ,  a ww1 battlefield BREAK (best quality, masterpiece:1.4) ,  a battlemech BREAK (best quality, masterpiece:1.4) ,  a sherman tank
  0%|                                                                                           | 0/20 [00:00<?, ?it/s]
gradio call: TypeError
┌───────────────────────────────────────── Traceback (most recent call last) ─────────────────────────────────────────┐
│ C:\StableDiffusion\vladmandic\modules\call_queue.py:58 in f                                                         │
│                                                                                                                     │
│    57 │   │   │   │   pr.enable()                                                                                   │
│ >  58 │   │   │   res = list(func(*args, **kwargs))                                                                 │
│    59 │   │   │   if shared.cmd_opts.profile:                                                                       │
│                                                                                                                     │
│ C:\StableDiffusion\vladmandic\modules\call_queue.py:38 in f                                                         │
│                                                                                                                     │
│    37 │   │   │   try:                                                                                              │
│ >  38 │   │   │   │   res = func(*args, **kwargs)                                                                   │
│    39 │   │   │   finally:                                                                                          │
│                                                                                                                     │
│                                              ... 33 frames hidden ...                                               │
│                                                                                                                     │
│ C:\StableDiffusion\vladmandic\extensions\sd-webui-regional-prompter\scripts\attention.py:367 in forward             │
│                                                                                                                     │
│   366 │   │   │   elif self.cells:                                                                                  │
│ > 367 │   │   │   │   ox = matsepcalc(x, contexts, mask, True, 1)                                                   │
│   368 │   │   │   else:                                                                                             │
│                                                                                                                     │
│ C:\StableDiffusion\vladmandic\extensions\sd-webui-regional-prompter\scripts\attention.py:141 in matsepcalc          │
│                                                                                                                     │
│   140 │   │   │   if self.usebase:                                                                                  │
│ > 141 │   │   │   │   context = contexts[:,tll[i][0] * TOKENSCON:tll[i][1] * TOKENSCON,:]                           │
│   142 │   │   │   │   # SBM Controlnet sends extra conds at the end of context, apply it to al                      │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
TypeError: slice indices must be integers or None or have an __index__ method
```
![txt2img_settings](https://user-images.githubusercontent.com/131223696/235525389-3be3355f-eac4-4c20-b88b-5cf35b0e1f10.PNG)
![regionalprompter_settings](https://user-images.githubusercontent.com/131223696/235525393-296eacb5-6828-439f-8575-4ccdebe160ca.PNG)

**Comment by AxelFar94** [comment]
I'm getting the same error

**Comment by artifabrian** [comment]
Likewise! The issue is because those objects are lists and can't be used to slice I believe:

```
e.g.
tll[i][0]: [0, 1]
tll[i][1]: [1, 2]
```

**Comment by artifabrian** [comment]
A temporary workaround until the bug is fixed and properly investigated is to use the Latent mode instead of Attention as the bug is just on the Attention hook forward.

**Comment by Symbiomatrix** [comment]
@artifabrian Thanks for checking. Tll is supposed to be a list of start/end ranges of each region's relevant contexts. Is it perchance wrapped in another list layer?

**Comment by hako-mikan** [comment]
My mistake. It seems I uploaded a different file than I intended.

**Comment by crazygonuts** [comment]
fixed for me on 5954877

**Comment by hako-mikan** [comment]
Thanks reporting.
